src/main/java/uk/gov/companieshouse/presenter/account/service/PresenterAccountService.java

```diff
@@ -8,6 +8,9 @@
 import uk.gov.companieshouse.presenter.account.model.mapper.presenter.account.mapper.PresenterAccountDetailsMapper;
 import uk.gov.companieshouse.presenter.account.model.mapper.presenter.account.mapper.base.Mapper;
 import uk.gov.companieshouse.presenter.account.model.request.presenter.account.PresenterRequest;
+import uk.gov.companieshouse.presenter.account.repository.PresenterAccountRepository;
+
+import java.util.UUID;
```

Comment:
The imports should be in alphabetical order (this will fail sonar once we install it)

Reply:
Fixed the review comment.